src/ghost/ghost-post.service.ts

```diff
@@ -113,4 +129,57 @@ export class GhostPostService {
 
         return await this.postService.deleteByApId(apId, account);
     }
+
+    async createGhostPost(
+        account: Account,
+        data: GhostPost,
+    ): Promise<Result<Post, GhostPostError>> {
+        const existingPost = await this.getApIdForGhostPost(data.uuid);
+
+        if (existingPost) {
+            return error('post-already-exists');
+        }
+
+        const postResult = await this.postService.handleIncomingGhostPost(
```

Comment:
Seeming though this is the Ghost post service, shouldn't the Ghost specific logic be here instead of in `handleIncomingGhostPost`? Do we even need `postService.handleIncomingGhostPost` anymore if we could handle everything Ghost specific here?